web-common/src/features/dashboards/time-controls/TimeRangeNameSelector.svelte

```diff
@@ -0,0 +1,101 @@
+<script lang="ts">
```

Comment:
I think this file may be redundant and we're using `TimeRangeSelector.svelte` now.